### DISCUSSION TITLE
Converstion to Playwright

### DISCUSSION THREAD

**Discussion by filcole** [Capgemini/powerapps-specflow-bindings#170] [Ideas]
These BDD bindings are great, as is the data generation and dynamics data.  It would be great if it these bindings were migrated to support playwright (possibly on top of cucumber.js).

Is this something the capgem team have considered or would be interested in?   Is removing the dependency on SpecFlow and EasyRepro practical?

p.s.  I know Microsoft have '[Test-Engine](https://powerapps.microsoft.com/en-us/blog/introducing-test-engine-an-open-platform-for-automated-testing-of-canvas-apps/)' in the works, but this may take a while to get out of experimental mode and gain support for model driven apps.

**Comment by tdashworth** [comment]
Hey @filcole, it is something we have discussed but it's unlikely we would refactor this package. That is because the fundamental architecture of this project was to expose the existing API of EasyRepro which uses Selenium. Therefore, a Playwright based solution would be something new and currently we don't have the capacity.
